### ISSUE TITLE
MS Monaco Consideration

### ISSUE THREAD

**Issue by kenashworth** [WordPress/better-code-editing#22]
https://github.com/Microsoft/monaco-editor/labels/accessibility
https://github.com/Microsoft/vscode/labels/accessibility

The Monaco Editor seems to be WCAG compliant, and both teams track against accessibility (Monaco is derived from VSCode). It looks harder to integrate (because I've been working with CodeMirror for 6 years) and I think is a larger codebase, but being actually WCAG compliant might have a snowball's chance of not getting shot down.

**Comment by kenashworth** [comment]
https://github.com/Microsoft/monaco-editor/wiki/Monaco-Editor-Accessibility-Guide
https://github.com/Microsoft/monaco-editor/wiki/Accessibility-Guide-for-Integrators

Some guides for Accessibility when using and integrating

**Comment by kenashworth** [comment]
Drawback is it doesn't work on mobile(?)

**Comment by melchoyce** [comment]
@westonruter Any way we could figure out what they're doing to make it accessible, and then port that over to CodeMirror?

**Comment by westonruter** [comment]
I think CodeMirror is going to win out regardless because of its much longer history (since 2011 instead of 2016), number of contributors (586 vs 17)  and releases (109 vs 13) and stability (5.28.0 vs 0.10.0). So I think that CodeMirror should just look itself at any benefits that MS Monaco is doing. For our part, we _are_ using CodeMirror's `contentEditable` mode which _is_ better for compatibility. I've commented on the CodeMirror issue to raise the question about what it would take to address any accessibility defects: https://github.com/codemirror/CodeMirror/issues/4604#issuecomment-324530352

Unless there is clear overall benefits of MS Monaco over CodeMirror, I'll close this.
